novawallet/Common/Extension/Foundation/Decimal+Conversion.swift

```diff
@@ -66,4 +66,74 @@ extension Decimal {
 
         return decimal2 / decimal1
     }
+
+    func greaterThanOrEqual(
```

Comment:
Why it is important to have that functions?

Reply:
The scale of compared values may differ. For example, if we compare 0.957449566 to 0.9575, we may need to round them to the same scale. A good example is referendum approval logic, where we compare fraction values to a more precise threshold value. The threshold value may appear the same when rounded to a lower scale but can be different when compared at a higher scale.